drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -536,7 +538,7 @@ static int hf_clock_enable(bool on, bool blocking)
 	struct device *clock;
 	static bool clock_requested;
 
-	clock = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL);
+	clock = device_get_binding(DT_INST_LABEL(0));
```

Comment:
Once more the "counter_nrfx_rtc" case.
```suggestion
	clock = device_get_binding(DT_PROP(DT_INST(0, nordic_nrf_clock), label));
```